src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/__tests__/ArtworkContextAuction.test.tsx

```diff
@@ -9,9 +9,7 @@ describe("ArtworkContextAuction", () => {
   const getWrapper = (props = ArtworkContextAuctionFixture) => {
     return mount(
       <RelayStubProvider>
-        <ArtworkContextAuctionFragmentContainer
-          artwork={props.artwork as any}
-        />
+        <ArtworkContextAuctionFragmentContainer viewer={props.viewer as any} />
```

Comment:
I have no clue why this spec is failing (the error output is hard to grok). I updated the fixture and signature here to just reflect the nesting change to `viewer`.